src/components/victory-area/victory-area.js

```diff
@@ -179,9 +179,19 @@ export default class VictoryArea extends React.Component {
     ]),
     /**
      * The samples prop specifies how many individual points to plot when plotting
+     * @param {Object} props The component of which the samples prop belongs to.
+     * @param {String} propName The name of the prop of which the function belongs to.
```

Comment:
This comment block ends get converted to a props description by our auto docs generation tool, so this function annotation can't stay here